nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/CLICommand.java

```diff
@@ -1212,15 +1203,24 @@ private String passwordName(ParamModel opt) {
     protected char[] readPassword(String prompt) {
         char[] pc = null;
 
-        try (ConsoleReader consoleReader = new ConsoleReader(System.in, System.out, null)) {
-            // Don't echo anything when reading
-            char echoCharacter = 0;
-            consoleReader.setEchoCharacter(echoCharacter);
+        char echoCharacter = 0;
+        LineReader lineReader = null;
+        try {
+            lineReader = LineReaderBuilder.builder()
+                    .terminal(new DumbTerminal(System.in, System.out)).build();
 
-            String line = consoleReader.readLine(prompt);
+            String line = lineReader.readLine(prompt, echoCharacter);
```

Comment:
Not convinced this works.
I built it locally and it still echoes out the password as you type it.